pkg/checkup/workspace.go

```diff
@@ -352,3 +352,23 @@ func (w *workspace) StartAndWaitCheckupJob(client *kubernetes.Clientset) error {
 
 	return nil
 }
+
+func (w *workspace) Teardown(client *kubernetes.Clientset) error {
+	if err := w.deleteNamespace(client); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (w *workspace) deleteNamespace(client *kubernetes.Clientset) error {
+	name := w.namespace.Name
+	if err := client.CoreV1().Namespaces().Delete(context.Background(), name, metav1.DeleteOptions{}); err != nil {
+		return err
+	}
+
+	log.Printf("Successfuly deleted namesapce: %q\n", name)
```

Comment:
nit: Why not using `%s`?
Also I think you should remove cap letter at the start of the message.